pages/dashboard.py

```diff
@@ -0,0 +1,21 @@
+from selenium.webdriver.chrome.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+
+from pages.base_page import BasePage
+
+
+class Dashboard(BasePage):
+    def __init__(self, driver: WebDriver):
+        super().__init__(driver)
+
+    def choose_product_list(self, name) -> None:
```

Comment:
Annotation will make this name argument more clear for other users of this code.